packages/lumx-react/src/components/date-picker/DatePickerField.test.tsx

```diff
@@ -49,19 +50,24 @@ const setup = ({ ...propsOverrides }: SetupProps = {}, shallowRendering: boolean
     const renderer: (el: ReactElement) => Wrapper = shallowRendering ? shallow : mount;
 
     const wrapper: Wrapper = renderer(<DatePickerField {...props} />);
+    const withDateObjectWrapper: Wrapper = renderer(
+        <DatePickerField {...props} value={new Date('January 18, 1970')} />,
+    );
 
     return {
         props,
         wrapper,
+        withDateObjectWrapper,
     };
 };
 
 describe(`<${DatePickerField.displayName}>`, () => {
     // 1. Test render via snapshot (default states of component).
     describe('Snapshots and structure', () => {
         it('should render correctly', () => {
-            const { wrapper } = setup();
+            const { wrapper, withDateObjectWrapper } = setup();
```

Comment:
This is not how we test our component. You should create another dedicated `it()` block and use the `setup()` method to override the value. You should not create a new wrapper.

Something like this:
```js
const { wrapper } = setup({ value: new Date('January 18, 1970') });
```